kalite/distributed/templates/distributed/base.html

```diff
@@ -53,26 +61,25 @@
             <script type="text/javascript" src="//cdn.crowdin.com/jipt/jipt.js"></script>
         {% endif %}
 
+        <!-- JS translation file statically included as a back-up, so that if the saved file below isn't available, we don't choke. -->
+        {% if current_language != "en" %}
+        <script type="text/javascript" src="{% static 'js/i18n/en.js' %}"></script>
+        {% endif %}
+
         <!-- JS translation files dynamically generated by django, then saved to the static file below -->
-        <script type="text/javascript" src="{% static 'js/i18n/'|add:request.language|add:'.js' True %}"></script>
+        <script type="text/javascript" src="{% static 'js/i18n/'|add:request.language|add:'.js' %}"></script>
```

Comment:
Having Django template variables inside the compression blocks makes them not work in a reliable enough way for offline compression, so I have left these dynamically loaded files out for now.